src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/ContextClasses.cs

```diff
@@ -11,7 +10,7 @@ namespace System.Text.Json.SourceGeneration.Tests
     public interface ITestContext
     {
         public JsonSourceGenerationMode JsonSourceGenerationMode { get; }
-        public bool IsIncludeFieldsEnabled => GetType().GetCustomAttribute<JsonSourceGenerationOptionsAttribute>()?.IncludeFields ?? false;
+        public bool IsIncludeFieldsEnabled { get; }
```

Comment:
IIUC we don't ship that attribute in netfx?

Reply:
the problem is default interface implementation, I was getting compile errors telling me that it can't be used on this target framework